de.metas.contracts/src/main/java/de/metas/contracts/subscription/process/C_SubscriptionProgress_ChangeRecipient.java

```diff
@@ -50,6 +50,9 @@
 	@Param(parameterName = "DropShip_User_ID", mandatory = false)
 	private int DropShip_User_ID;
 
+	@Param(parameterName = "IsPermanentRecipient", mandatory = true)
```

Comment:
watching the full file, i saw that dateTo is aparently still mandatory..
how does it work? maybe you should make it non-mandatory, or set a default and set the param's display logic (if possible)?

Reply:
@metas-ts if the IsPermanentRecipient is set on yes, that entire field is ignored.
DateFrom and DateTo come from a date field range parameter, so are all the time field up.
But, what I could do(not sure if is working in webui this) is to add a display logic for the dates field, which will be hidden if the user ticks IsPermanentRecipient flag.
WDYT?